packages/flutter_tools/test/general.shard/base/command_help_test.dart

```diff
@@ -0,0 +1,171 @@
+  expect(CommandHelp.P.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.R.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.S.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.U.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.a.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.d.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.h.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.i.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.o.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.p.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.q.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.r.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.s.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.t.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.w.toString().length, lessThanOrEqualTo(expectedWidth));
+  expect(CommandHelp.z.toString().length, lessThanOrEqualTo(expectedWidth));
+};
+
+void main() {
+  group('CommandHelp', () {
+    group('toString', () {
+
+      final MockPlatform platform = MockPlatform();
+
+      testUsingContext('should have a bold command key', () async {
+        when(platform.stdoutSupportsAnsi).thenReturn(true);
+
+        expect(CommandHelp.L.toString(), startsWith('\x1B[1mL\x1B[22m'));
+        expect(CommandHelp.P.toString(), startsWith('\x1B[1mP\x1B[22m'));
+        expect(CommandHelp.R.toString(), startsWith('\x1B[1mR\x1B[22m'));
+        expect(CommandHelp.S.toString(), startsWith('\x1B[1mS\x1B[22m'));
+        expect(CommandHelp.U.toString(), startsWith('\x1B[1mU\x1B[22m'));
+        expect(CommandHelp.a.toString(), startsWith('\x1B[1ma\x1B[22m'));
+        expect(CommandHelp.d.toString(), startsWith('\x1B[1md\x1B[22m'));
+        expect(CommandHelp.h.toString(), startsWith('\x1B[1mh\x1B[22m'));
+        expect(CommandHelp.i.toString(), startsWith('\x1B[1mi\x1B[22m'));
+        expect(CommandHelp.o.toString(), startsWith('\x1B[1mo\x1B[22m'));
+        expect(CommandHelp.p.toString(), startsWith('\x1B[1mp\x1B[22m'));
+        expect(CommandHelp.q.toString(), startsWith('\x1B[1mq\x1B[22m'));
+        expect(CommandHelp.r.toString(), startsWith('\x1B[1mr\x1B[22m'));
+        expect(CommandHelp.s.toString(), startsWith('\x1B[1ms\x1B[22m'));
+        expect(CommandHelp.t.toString(), startsWith('\x1B[1mt\x1B[22m'));
+        expect(CommandHelp.w.toString(), startsWith('\x1B[1mw\x1B[22m'));
+        expect(CommandHelp.z.toString(), startsWith('\x1B[1mz\x1B[22m'));
+      }, overrides: <Type, Generator>{
+        OutputPreferences: () => OutputPreferences(wrapColumn: maxLineWidth),
+        Platform: () => platform,
```

Comment:
Create mockPlatform here instead of in the group. This prevents mocks from previous test runs from leaking into other tests

Reply:
Yes, of course! Then the instance I get from the test context is going to be the mock. Thanks!